projects/plugins/crm/modules/automations/jpcrm-automations-init.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
**Request**: I assume that this was converted from a class to function due to "the magical logic that loads the init file", but would it be cleaner if we separated this into two files?

1. `class-admin-page.php`: Contains an `Admin_Page` class with 90% of the code from this file.  
2. `jpcrm-automations-init.php` Basically just calls `Admin_Page->init()` together with the feature flag (there might be some magical loading logic I'm forgetting).

By doing the above, we can also init other classes when we e.g. start to bootstrap the automations engine and it doesn't have to be mixed together with admin page related code

Reply:
I made it functions instead of a class because there is no state here. So in the onboarding wizard there is a singleton pattern and accessing the singleton causes all of the init logic to run, but it is all functions with no state so I decided to just make this functions.

Until there is state I'd prefer to keep things as functions just so the semantics of the code match the functionailty, but I can certainly begin splitting out this logic into more files.

Reply:
I split out all the admin page init logic into `admin-page-init.php`.